Resources/Prototypes/Nyanotrasen/Entities/Clothing/OuterClothing/wintercoats.yml

```diff
@@ -39,6 +39,8 @@
   name: mail carrier's winter coat
   description: It'll keep away the cold but not the dogs.
   components:
+    - type: ToggleableClothing
+     clothingPrototype: ClothingHeadHatHoodWinterMailCarrier #DeltaV - fixing the fact that it has no hood
```

Comment:
add 1 space before clothingPrototype and move the comment to the line above

Reply:
i did it 👍